server/datastore/mysql/software.go

```diff
@@ -596,11 +598,9 @@ func countSoftwareDB(
 	return count, nil
 }
 
-func (ds *Datastore) LoadHostSoftware(ctx context.Context, host *fleet.Host, includeCVEScores bool) error {
-	opts := fleet.SoftwareListOptions{
-		HostID:           &host.ID,
-		IncludeCVEScores: includeCVEScores,
-	}
+func (ds *Datastore) LoadHostSoftware(ctx context.Context, host *fleet.Host, opts fleet.SoftwareListOptions, includeCVEScores bool) error {
```

Comment:
I am a bit confused why this function now accepts both `host`, `includeCVEScores` **and** `opts`. It's not clear how these arguments would conflict without reading the code inside the function. Currently, the only usage is in `Service.getHostDetails` which just passes the zero value for opts.

Reply:
Removed `SoftwareListOptions` from `LoadHostSoftware`